client/public/loginPage/loginScript.js

```diff
@@ -0,0 +1,45 @@
+const loginForm = document.getElementById("loginFrm");
+const emailInpt = document.getElementById("emailInpt");
+const passwordInpt = document.getElementById("passwordInpt");
+
+loginForm.addEventListener("submit", async (e) => {
+  e.preventDefault();
+  const userInfo = {
+    email: loginForm.email.value,
+    password: loginForm.password.value,
+  };
+  try {
+    if (userInfo.email !== null || userInfo.password !== null) {
+      const data = await login(userInfo);
+      if (data.isLoggedIn) {
+        if (data.role === "manager") {
+          window.location.replace("http://localhost:4000/public/dashboard");
```

Comment:
`window.location.replace("http://localhost:4000/public/dashboard");`
do you do that to redirect the user?
seems like a workaround and also, the user can control the flow simply by changing this code and accessing the page. I would have done all these checks on the backend side and redirected the user accordingly.

Reply:
@DaveAvigad here. i dont know how to do this in the backend